README.md

```diff
@@ -1 +1,9 @@
-# hw01
\ No newline at end of file
+# hw01
+## Biography
+My name is Carol Medina. I am a psychology graduate student at the University of Chicago (https://psychology.uchicago.edu/directory/carol-medina). I live in Hyde Park with my cat, **Luna**. Some of my favorite things to do include: 
```

Comment:
This link should look like this:
[link](https://psychology.uchicago.edu/directory/carol-medina)
and this is how you do it:
`[link](https://psychology.uchicago.edu/directory/carol-medina)`